### DISCUSSION TITLE
Is it possible to get activeSlide styling  immediately on scroll in Wheel Gesture Plugin?

### DISCUSSION THREAD

**Discussion by sushantgwr87** [davidjerleke/embla-carousel#505] [Help] [answered]
Hello, I am trying to create a vertical carousel which is scrollable. I used Wheel Gesture Plugin to achieve it.

Issue is  active slide is not updated immediately and active-slide styling is not applied until user stops scrolling. I want to show next active slide as soon as user scrolls.

I am thinking to add snap scroll but don't know how to add it.

Any help will be appreciated.

Demo:
https://codesandbox.io/s/onwheelgesturecarousel-l5ww8x

**Comment by davidjerleke** [comment, marked as answer]
Hi @sushantgwr87,

Yes. Use the `select` and `pointerUp` event instead of `scroll`. [Example here](https://codesandbox.io/s/onwheelgesturecarousel-forked-2jkhrl?file=/src/Carousel.js).  I also added `skipSnaps: true` to the carousel options because with that small slide heights it feels weird when it's enforcing scrolling a single slide when the wheel is rolled vigorously. 

Best,
David

**Reply by sushantgwr87** [reply]
Thanks for so quick answer. Yeah Slide height is too small as it is text only carousel.

Also thanks for this amazing package 😁
